expressServer/scripts/deleteAllPasswords.js

```diff
@@ -0,0 +1,36 @@
+const dotenv = require('dotenv');
+const { connectDB, closeDB } = require('../database/database.js');
+const { User } = require('../database/models/User.js');
+
+dotenv.config();
+
+async function encryptAllUserPasswords() {
+    const batchSize = 1000;
+    let totalProcessed = 0;
+    let skip = 0;
+
+    try {
+        await connectDB();
+
+        while (true) {
+            const users = await User.find({}).skip(skip).limit(batchSize);
+            if (users.length === 0) break;
+
+            for (let user of users) {
+
+                delete user.password;
+
+                await user.save();
+                totalProcessed++;
+            }
+            console.log(`Processed ${totalProcessed} users.`);
+            skip += batchSize;
+        }
+    } catch (error) {
+        console.error('Error deleting user passwords:', error);
+    } finally {
+        await closeDB();
+    }
+}
+
+encryptAllUserPasswords();
```

Comment:
Your script does not wait to finish, you need to include then, catch and finally clauses.